tools/cpp/unix_cc_configure.bzl

```diff
@@ -189,6 +189,17 @@ def _is_gold_supported(repository_ctx, cc):
     ])
     return result.return_code == 0
 
+def _is_fission_supported(repository_ctx, cc):
+    """Checks if `DebugFission` is supported by the C compiler."""
+    result = repository_ctx.execute([
+        cc,
+        "-gsplit-dwarf",
+        "-g",
+        "-c",
```

Comment:
You probably need to add -o /dev/null to avoid a non-sensical output file in the tools tree.